examples/guess-the-sketch/matchmaker/frontend_k8s.yaml

```diff
@@ -36,6 +45,9 @@ spec:
             - containerPort: 8001
           command: ["/app/frontend"]
           args: ["-showConsentPage=false"]
+          env:
+          - name: OM_CORE_ADDRESS
+            value: "https://om-core-976869741551.us-central1.run.app"
```

Comment:
This looks like a hard coded address. I assume it will not work outside your project? Can we dynamically pass it in?

Reply:
Same as the comment above.